tests/phpunit/AutoReview/Makefile/MakefileTest.php

```diff
@@ -323,7 +323,7 @@ static function (array $targetSet): bool {
 
                 return strpos($target, 'test') === 0
                     && substr($target, -7) === '-docker'
-                    && ([] === $dependencies
+                    && ($dependencies === []
```

Comment:
These changes could have went to another PR.

Reply:
Absolutely agree, didn't notice it after running `make cs`. Done and merged in separate PR: https://github.com/infection/infection/pull/1279